### DISCUSSION TITLE
Tailwinds Components for Bootstrap

### DISCUSSION THREAD

**Discussion by ErikThiart** [twbs/bootstrap#40710] [Ideas]
Hello guys, is there no way we could shift focus and maybe try to create a repository like Tailwinds have of high quality finished components one can use: https://tailwindui.com/components

I am sure most people would be willing to buy a lifetime license for access to high end bootstrap compatible components that matches that style.

My current issue with bootstrap is the design language (visual) feels very dated, whereas tailwinds' components feels very modern. 

I know in theory bootstrap can be made look modern, but not without a lot of expertise, so my suggestion is for the Bootstrap Foundation to focus on adding something on par with Tailwind's component library.

Thoughts?

**Comment by ErikThiart** [comment]
Here is a prime example of why I think this needs to be a Bootstrap Foundation level thing. 

https://github.com/orgs/twbs/discussions/35385#discussioncomment-10285047

The above guy had the perfect idea, exactly what we need, just make sure it is maintained.

**Comment by ErikThiart** [comment]
At the very least we should build out on these component pages and add the code snippets for each

https://getbootstrap.com/docs/5.3/examples/jumbotrons/

Right now it feels very... afterthought and not polished.

**Comment by supercharles76** [comment]
Hi ErikThiart,
I found this project called [NinjaBootstrap UI](https://ui.bootstrap.ninja/), which is a collection of Tailwind components designed for Bootstrap. To use them, you need to use their modified version of Bootstrap 5 (which you can find here) https://bootstrap.ninja/ninjabootstrap/ to get some very powerful and useful utility classes that I felt were missing from Bootstrap 5.

**Comment by ErikThiart** [comment]
Good starting point, thank you.
